### PR TITLE
[FIX] hr_holidays: show correct remaining leaves value

### PR DESCRIPTION
**Pull request by Bertrand2**
taskID 2971148

### CODE REVIEW

File: addons/hr_holidays/models/hr_employee.py [hunk 1]
```diff
@@ -98,20 +98,17 @@ def _compute_allocation_count(self):
...
-            result = results_leave.get(employee.id)
-            leaves_taken = float_round(result['number_of_days'], precision_digits=2) if result else 0.0
-            employee.allocation_remaining_display = "%g" % float_round(employee.allocation_count - leaves_taken, precision_digits=2)
+            employee_remaining_leaves = 0
+            for leave_type in leaves_taken[employee.id]:
+                if leave_type.requires_allocation == 'no':
+                    continue
+                for allocation in leaves_taken[employee.id][leave_type]:
+                    if allocation:
+                        employee_remaining_leaves += leaves_taken[employee.id][leave_type][allocation]['virtual_remaining_leaves']
```
Comment (phwa-odoo) [inline addons/hr_holidays/models/hr_employee.py:110]:
```suggestion
                       virtual_remaining_leaves = leaves_taken[employee.id][leave_type][allocation]['virtual_remaining_leaves']
                        employee_remaining_leaves += virtual_remaining_leaves if leave_type.request_unit in ['day', 'half_day'] else virtual_remaining_leaves / (employee.resource_calendar_id.hours_per_day or HOURS_PER_DAY)
```
`HOURS_PER_DAY` comes from the following import `from odoo.addons.resource.models.resource import HOURS_PER_DAY`
`_get_employee_days_per_allocation` returns the virtual_remaining_leaves in hours or days depending on the request_unit of the leave type. I can be seen on Mitchell Admin's profile who has 148 remaining days out of 39.

File: addons/hr_holidays/models/hr_employee.py [hunk 2]
```diff
@@ -98,20 +98,17 @@ def _compute_allocation_count(self):
 
     def _compute_allocation_remaining_display(self):
         current_date = datetime.date.today()
```
Comment (phwa-odoo) [inline addons/hr_holidays/models/hr_employee.py:100]: No longer needed

**Review by phwa-odoo — COMMENTED** [review]
Commit message should be more descriptive

**Comment by C3POdoo** [comment]
Dear @Bertrand2,

Thank you for your contribution but we are closing it due to inactivity.
We only support the last 3 stable versions so no longer accepts patches into this branch.

We apology if we could not look at your request in time.
If the contribution still makes sense, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

_This is an automated message._

**Pull request closed without merging**